llama.cpp

```diff
@@ -4941,7 +4941,8 @@ static bool llm_load_tensors(
                     // output
                     {
                         model.output_norm = ml.create_tensor(ctx_output,       tn(LLM_TENSOR_OUTPUT_NORM, "weight"), {n_embd});
-                        if (model.arch != LLM_ARCH_MINICPM){
+                        //if (model.arch != LLM_ARCH_MINICPM){
+                        {
```

Comment:
Is this supposed to be commented out? 
If so, should the code block removed? (it now has no if)